cpp/src/arrow/dataset/file_rados_parquet.h

```diff
@@ -146,8 +159,10 @@ class ARROW_DS_EXPORT RadosParquetFileFormat : public ParquetFileFormat {
 
   Result<bool> IsSupported(const FileSource& source) const override { return true; }
 
+  /// \brief Return the schema of the file if possible.
   Result<std::shared_ptr<Schema>> Inspect(const FileSource& source) const override;
 
+  /// \brief Open a file for scanning.
```

Comment:
This method does not particularly open a file. We can just say "Scan a file fragment"

Reply:
Done.